pyproject.toml

```diff
@@ -118,6 +117,15 @@ test = { cmd = "test", help = "Run pytest" }
 [tool.taskipy.settings]
 runner = "pdm run"
 
+[tool.pdm.build]
+includes = [
+    "disnake/",
```

Comment:
Does this do anything? Seems like `tests/` is also included, for instance (which is correct).